src/main/java/io/github/enkarin/bookcrossing/books/service/BookService.java

```diff
@@ -31,27 +29,26 @@ public Optional<BookResponse> saveBook(final BookDto bookDTO, final String login
         if (book.isEmpty()) {
             return Optional.empty();
         }
-        book = Optional.of(bookRepository.save(book.get()));
-        final BookResponse response = new BookResponse(book.get());
+        final BookResponse response = new BookResponse(modelMapper.map(
+                bookRepository.save(book.get()), BookModelDto.class));
         return Optional.of(response);
     }
 
-    public List<Book> findBookForOwner(final String login) {
+    public Optional<BookListResponse> findBookForOwner(final String login) {
```

Comment:
It's better to return List<BookModelDto>

Reply:
And result List<> may be empty

Reply:
Made for documentation, since it is impossible to specify a List there

Reply:
Changed to List